docs/usage/sagas/azure-table.md

```diff
@@ -33,8 +33,10 @@ container.AddMassTransit(cfg =>
         .AzureTableRepository(endpointUri, key, r =>
         {
            cfg.ConnectionFactory(() => cloudTable);
+           // optionally specify the key formatter to use: 
+           // cfg.KeyFormatter(() => new PartitionSagaKeyFormatter<OrderState>(typeof(OrderState).Name)))
```

Comment:
Rather than comment it out - we could have another configuration section in the docs below. Allows us to keep the syntax highlighting & focus the docs a little more on the ability of the key formatter

Reply:
Is it also worth adding reference to the documentation you mentioned in the issue which highlights the issue of busy partitions?

Reply:
updated